File: Modul-6/C/PRAK601-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int x, y, Matriks, Baris, Kolom;
    scanf("%d %d", &Baris, &Kolom);

    int Num[Baris][Kolom];
    for (x = 0; x < Baris; x++)
    {
        for (y = 0; y < Kolom; y++)
        {
            scanf("%d", &Matriks);
            Num[x][y] = Matriks;
        }
    }

    for (x = 0; x < Baris; x++)
    {
        for (y = 0; y < Kolom; y++)
        {
            printf("%d ", Num[x][y]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul-6/Python/PRAK601-2310817320007-ZahraNabila.py
Kolom, Baris = map(int, input().split())
Num = list(map(int, input().split()))
Matriks = []
x = 0

for y in range (Kolom):
    Matriks.append(Num[x : x + Baris])
    x = x + Baris

for Kolom in Matriks:
    for Num in Kolom:
        print(Num, end = " " )
    print()